clang/test/SemaSYCL/sycl-device-num_simd_work_items-template.cpp

```diff
@@ -5,8 +5,9 @@
 // Test that checks wrong function template instantiation and ensures that the type
 // is checked properly when instantiating from the template definition.
 template <typename Ty>
-// expected-error@+2{{integral constant expression must have integral or unscoped enumeration type, not 'S'}}
-// expected-error@+1{{integral constant expression must have integral or unscoped enumeration type, not 'float'}}
+// expected-error@+3{{integral constant expression must have integral or unscoped enumeration type, not 'S'}}
+// expected-error@+2{{integral constant expression must have integral or unscoped enumeration type, not 'float'}}
+// expected-error@+1{{'num_simd_work_items' attribute requires a positive integral compile time constant expression}}
```

Comment:
I see. This is because `func<int>` is same as `func<0>` ?

Reply:
Correct; in the expansion, the `int{}` is a constant expression whose value is `0`.